arangod/Cluster/SynchronizeShard.cpp

```diff
@@ -1017,11 +913,12 @@ bool SynchronizeShard::first() {
       auto lastTick =
         arangodb::basics::VelocyPackHelper::getNumericValue<TRI_voc_tick_t>(
           sy, LAST_LOG_TICK, 0);
-      VPackBuilder builder;
 
       ResultT<TRI_voc_tick_t> tickResult =
-        catchupWithReadLock(ep, *collection, clientId, leader, lastTick, builder);
+          catchupWithReadLock(ep, *collection, clientId, leader, lastTick, tailingSyncer);
```

Comment:
With the check suggested above, this call is safe, since the called function assumes that `tailingSyncer` is not a nullptr.